minecraft_servers/lostify/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "server_name": "lostify",
+  "nice_name": "Lostify Servernetzwerk",
+  "direct_ip": "lostify.net",
+  "server_wildcards": [
+    "%.lostify.net"
+  ],
+  "social": {
+    "web": "https://lostify.net",
+    "web_shop": "https://shop.lostify.net",
+    "web_support": "https://forum.lostify.net",
```

Comment:
Please add your faq or a direct support page here

Reply:
I have added a direct link to our forum support